pkg/api/validation/dynakube/modules.go

```diff
@@ -24,10 +25,14 @@ func isOneAgentModuleDisabled(_ context.Context, v *Validator, dk *dynakube.Dyna
 }
 
 func isActiveGateModuleDisabled(_ context.Context, v *Validator, dk *dynakube.DynaKube) string {
-	if dk.ActiveGate().IsEnabled() && !v.modules.ActiveGate && !v.modules.KSPM {
+	if dk.ActiveGate().IsEnabled() && !v.modules.ActiveGate {
 		return errorActiveGateModuleDisabled
 	}
 
+	if dk.ActiveGate().IsKubernetesMonitoringEnabled() && !v.modules.KubernetesMonitoring && !v.modules.KSPM {
+		return errorKubernetesMonitoringModuleDisabled
+	}
```

Comment:
this is the "questionable" validation, but it does the same as before. 

as before the main purpose of `v.modules.ActiveGate` (in helm `Values.rbac.activeGate) was to limit the kubemon related RBACs (because there are no other RBACs for the AG)

now it is a bit more fine grained.